http_handler.go

```diff
@@ -43,23 +46,35 @@ func Trace(handler http.Handler) http.Handler {
 // necessary.
 
 // In addition to tracing capabilities this function also provides graceful
-// shutdown of the server. When a SIGTERM signal is received by the server
-// it will stop accepting new connections but it will keep it running until
-// all the existing connections have been closed. It will kill the server
+// shutdown of the server. When a SIGTERM or SIGINT signal is received by the
+// server it will stop accepting new connections but it will keep it running
+// until all the existing connections have been closed. It will kill the server
 // after a default timeout of 25 seconds if there are still any pending
 // connections.
 
-func ListenAndServe(addr string, handler http.Handler) error {
+func ListenAndServe(addr string, handler http.Handler) {
```

Comment:
Yes, now it will compile, but it is not very useful to have a function that returns an error that can never be anything but nil.